go-apps/meep-app-enablement/server/app-support/app-support.go

```diff
@@ -36,10 +38,11 @@ import (
 )
 
 const mappsupportBasePath = "/mec_app_support/v1/"
-const mappsupportKey = "mec-app-support"
+const mappsupportKey = "as-sub"
```

Comment:
The name is slightly misleading here. "as-sub" refers to the subscriptions key rather than a mec-app-support base key. Suggest renaming variable to "subscriptionsKey".

Reply:
changed the key to be "as" only if other resources are added in the future.
For now, only subscriptions can be created but keeping it so it is more generic